tensorboardX/writer.py

```diff
@@ -351,7 +352,7 @@ def add_hparams(self, hparam_dict=None, metric_dict=None):
             raise TypeError('hparam_dict and metric_dict should be dictionary.')
         exp, ssi, sei = hparams(hparam_dict, metric_dict)
 
-        with SummaryWriter(logdir=os.path.join(self.file_writer.get_logdir(), str(time.time()))) as w_hp:
+        with SummaryWriter(logdir=os.path.join(self.file_writer.get_logdir(), name) as w_hp:
```

Comment:
@clefourrier I guess lanpa referrs to the missing parenthesis in the end that closes the SummaryWriter call. Thanks for your commit, I really like it - already wanted to get it done myself before I found this PR.

Reply:
Thanks a lot, that was hastily copy pasted. Fixed it in new commit.